cmake/scripts/freebsd/ArchSetup.cmake

```diff
@@ -1,4 +1,7 @@
 set(ARCH_DEFINES -D_LINUX -DTARGET_POSIX -DTARGET_FREEBSD)
+if(CORE_PLATFORM_NAME_LC STREQUAL rbpi)
+  list(APPEND ARCH_DEFINES -D_ARMEL -DTARGET_RASPBERRY_PI)
```

Comment:
would be better fit into `cmake/platform/linux/rbpi.cmake`, like it is already done for aml and gbm

Reply:
meant `cmake/platform/freebsd/rbpi.cmake`